packages/edit-site/src/components/block-editor/site-editor-canvas.js

```diff
@@ -59,12 +63,25 @@ export default function SiteEditorCanvas() {
 	const isTemplateTypeNavigation = templateType === NAVIGATION_POST_TYPE;
 	const isNavigationFocusMode = isTemplateTypeNavigation && isFocusMode;
 	const forceFullHeight = isNavigationFocusMode;
+	const [ containerHeight, setContainerHeight ] = useState();
+	const containerRef = useRefEffect(
```

Comment:
I expect this is not ideal. 

It's a lot of faffery just to passing down the iframe container's height.

Maybe we pass down the ref? 🤷🏻